From bd76be12abfff4427231567404b41843417e6332 Mon Sep 17 00:00:00 2001
From: "Paul W. Frields" <stickster@gmail.com>
Date: Fri, 31 Jul 2015 15:36:14 -0400
Subject: [PATCH] Use freedesktop.org standard icon name

---
 src/pavucontrol.glade | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/pavucontrol.glade b/src/pavucontrol.glade
index 219f5a2..5efd4b3 100644
--- a/src/pavucontrol.glade
+++ b/src/pavucontrol.glade
@@ -312,7 +312,7 @@
                               <object class="GtkImage" id="image18">
                                 <property name="visible">True</property>
                                 <property name="can_focus">False</property>
-                                <property name="icon_name">stock_lock</property>
+                                <property name="icon_name">security-medium</property>
                                 <property name="icon-size">1</property>
                               </object>
                             </child>
@@ -1474,7 +1474,7 @@
                               <object class="GtkImage" id="image18">
                                 <property name="visible">True</property>
                                 <property name="can_focus">False</property>
-                                <property name="icon_name">stock_lock</property>
+                                <property name="icon_name">security-medium</property>
                                 <property name="icon-size">1</property>
                               </object>
                             </child>
-- 
2.4.3

